### ISSUE TITLE
Keyboard trap in carousel

### ISSUE THREAD

**Issue by joe-watkins** [twbs/bootstrap#19972]
Hello,

In v4 examples noticed a keyboard trap. @patrickhlauke 

**Steps to reproduce:** 
Visit: [Docs](http://v4-alpha.getbootstrap.com/components/carousel/#example) or [my CodePen test](http://codepen.io/joe-watkins/full/WxeYpY/) and tab into interface. Tab to carousel and notice focus is stuck on back pager. `<a class="left carousel-control">..</a>` Keyboard only user is unable to move focus to other focusable items.

**Expected Result:**
User should be able to freely move throught the document using only the keyboard

**Actual Result:**
User is not able to move throughout the document and is stuck on one element when focus is sent to it.

**Comment by twbs-lmvtfy** [comment]
Hi @joe-watkins!

You appear to have posted a live example (http://codepen.io/joe-watkins/pen/WxeYpY.html), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 18, column 3: [E041](https://github.com/twbs/bootlint/wiki/E041): `.carousel-inner` must have exactly one `.item.active` child.
* line 42, column 4: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by patrickhlauke** [comment]
Yup, carousel's broken in that respect.

**Comment by joe-watkins** [comment]
@patrickhlauke The `event.preventDefault()` on line 232 in carousel.js appears to be the culprit here. I killed that and that freed things up: http://s.codepen.io/joe-watkins/debug/JKjKXY
